core/src/main/java/bio/terra/pearl/core/service/datarepo/DataRepoExportUtils.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.pearl.core.service.datarepo;
+
+import bio.terra.datarepo.model.TableDataType;
+import bio.terra.pearl.core.service.export.formatters.DataValueExportType;
+
+public class DataRepoExportUtils {
+
+    //Converts a Juniper column type into a Data Repo column type
+    public static TableDataType juniperToDataRepoColumnType(DataValueExportType columnType) {
+        return switch(columnType) {
+            case DATE -> TableDataType.DATE;
+            case DATE_TIME -> TableDataType.DATETIME;
+            case STRING -> TableDataType.STRING;
+            case NUMBER -> TableDataType.INTEGER;
+            case BOOLEAN -> TableDataType.BOOLEAN;
+        };
+    }
+
+    //Converts a Juniper column name into a TDR-accepted column name
+    public static String juniperToDataRepoColumnName(String columnName) {
+        return columnName
+                .substring(0, Math.min(columnName.length(), 63)) //TDR accepts a maximum column name length of 64 chars
```

Comment:
What happens if this results in a non-unique column name? I'm less worried if it crashes; I'm much more worried if it silently does the wrong thing. I wonder if we need to do something like Windows used to do with munging long filenames to fit in the old "8.3" filename constraint.

Reply:
Very good question. TDR will reject the request:

```{"message":"Unable to create TDR dataset for study environment 529bd22a-af8c-4900-824d-bdf3f8247dd7. Error: {\"message\":\"Validation errors - see error details\",\"errorDetail\":[\"schema: 'DuplicateColumnNames' (Duplicate columns: foo)\"]}","statusCode":500}```

And just to be extra sure, I also verified that TDR still rejects the schema if two columns have the same name but different types (in case their definition of uniqueness is a bit more complex than just naming)

Reply:
I can also add a code comment about this.